src/dom/document-cloner.ts

```diff
@@ -127,9 +130,28 @@ export class DocumentCloner {
             return this.createStyleClone(node);
         }
 
+        if (isCustomElement(node)) {
+            return this.createCustomElementClone(node);
+        }
+
         return node.cloneNode(false) as HTMLElement;
     }
 
+    // custom elements are cloned as divs so the renderer knows how to render them
+    createCustomElementClone(node: HTMLElement): HTMLElement {
+        let customDiv = document.createElement('div');
```

Comment:
Something to consider is that `<div>` defaults to display `block` whereas custom-elements default to display: `inline`.
In some occasions this is not a problem, for example a child of a flex will always be considered a block, so you won't notice any difference.

Is it really necessary to convert custom-elements to a standard element?
I'm using Angular and the original library `1.0.0-rc.5`  didn't have any problem with custom-elements, the only problem was with the ones that use shadow dom encapsulation. When I turn to this fork, it supports shadow dom, although it did break height for one of the elements because of this inline vs block inconsistency. It was easy to fix with an onclone hook though.

In any case, switching to this fork was definitely an improvement for my use case, thanks for this contribution.
I should clarify that I'm not using _slots_ so I can't give feedback on that part.